### DISCUSSION TITLE
串口第一只能收到32个字符，但第二次就可能收完整

### DISCUSSION THREAD

**Discussion by 12255305** [itas109/CSerialPort#47] [Q&A]
使用CSerialPort异步接收数据，串口第一只能收到32个字符，但第二次就可能收完整，这是为什么？

**Comment by itas109** [comment]
类库可以保证数据不丢失，但是无法保证数据是否为完整的一帧数据。
确保数据是否为一帧数据需要开发者根据通信协议自己组成一帧数据。
